programs/address-lookup-table/Cargo.toml

```diff
@@ -19,11 +19,9 @@ serde = { workspace = true, features = ["derive"] }
 solana-frozen-abi = { workspace = true }
 solana-frozen-abi-macro = { workspace = true }
 solana-program = { workspace = true }
-thiserror = { workspace = true }
-
-[target.'cfg(not(target_os = "solana"))'.dependencies]
```

Comment:
not sure the reason of having a subset of dependencies. Now the program ID moved to sdk, I just move both to normal dependencies section. Will see if CI happy with it

Reply:
well, https://github.com/solana-labs/solana/pull/23700 intentionally removed `solana-sdk` dependency. So removing this `cfg` won't work.

Reply:
Pretty sure you want to revert this change. solana-sdk cannot be compiled for `target_os = "solana"`. instead you can include `solana-program = { workspace = true }` and grab the constants from  `solana_program::address_lookup_table_program::...` in this crate.